### PR TITLE
Added EN+RU languages support for feeds

### PR DESCRIPTION
**Pull request by ponjoru**

### CODE REVIEW

File: each/Entities/EntityNews.py [hunk 1]
```diff
@@ -98,12 +112,18 @@ def update_from_json(cls, data):
...
-                            _.text_RU = data['text_RU']
-
+                        if 'RU' in data['title']:
+                            _.title_RU = data['title']['RU']
+                        if 'EN' in data['title']:
+                            _.title_EN = data['title']['EN']
+                        if 'RU' in data['desc']:
+                            _.title_RU = data['desc']['RU']
+                        if 'EN' in data['desc']:
+                            _.title_EN = data['desc']['EN']
```
Comment (maik-nack) [inline each/Entities/EntityNews.py]: Change title_EN on desc_EN

File: each/Entities/EntityNews.py [hunk 2]
```diff
@@ -98,12 +112,18 @@ def update_from_json(cls, data):
...
+                        if 'RU' in data['text']:
+                            _.text_RU = data['text']['RU']
+                        if 'EN' in data['text']:
+                            _.title_EN = data['text']['EN']
```
Comment (maik-nack) [inline each/Entities/EntityNews.py]: Change title_EN on text_EN

File: each/Entities/EntityNews.py [hunk 3]
```diff
@@ -98,12 +112,18 @@ def update_from_json(cls, data):
...
-
-                        if 'text_RU' in data:
-                            _.text_RU = data['text_RU']
-
+                        if 'RU' in data['title']:
+                            _.title_RU = data['title']['RU']
+                        if 'EN' in data['title']:
+                            _.title_EN = data['title']['EN']
+                        if 'RU' in data['desc']:
+                            _.title_RU = data['desc']['RU']
```
[the remark this answers is not in the document]
Comment (ponjoru) [reply each/Entities/EntityNews.py]: fixed

File: each/Entities/EntityNews.py (hunk 1 above)
Comment (ponjoru) [reply each/Entities/EntityNews.py]: fixed

File: each/Entities/EntityNews.py (hunk 2 above)
Comment (ponjoru) [reply each/Entities/EntityNews.py]: fixed

**Pull request merged**